### PR TITLE
chore: added test cases to increase unit test coverage of atlas-core

### PR DESCRIPTION
**Pull request by joeylimd**
chore: added test cases to increase unit test coverage of atlas-core

Relates to: #16

### CODE REVIEW

File: runtime/core/pom.xml [hunk 1]
```diff
@@ -72,4 +72,35 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    
+    <build>
```
Comment (igarashitm) [inline runtime/core/pom.xml]: Why is this required? atlas-mode is already in dependency

Comment (joeylimd) [reply runtime/core/pom.xml]: Hi Tomo,
A jar file is needed to test AtlasUtil.findClassesFromJar method. copied atlas-model jar into target folder for Unit testing. Thanks

Comment (igarashitm) [reply runtime/core/pom.xml]: Ah OK... not really ideal to make POM fat with test only configuration though, I'd say OK for now.
